config/constants.php

```diff
@@ -6,4 +6,5 @@
     'ROLE_ADMIN' => 'Admin',
     'ROLE_BUSINESS' => 'Business',
     'ROLE_USER' => 'User',
+    'LENGTHTOKEN' => 'str_random(40)',
```

Comment:
[rank A] length of token should be contented the number of length, it's not means can get random function there.